nilearn/decoding/proximal_operators.py

```diff
@@ -7,7 +7,7 @@
 
 import numpy as np
 
-from .objective_functions import _div_id, _gradient_id, _tv_l1_from_gradient
+from .objective_functions import _div_id, _gradient_id, tv_l1_from_gradient
```

Comment:
because my detection algo is dumb and has false negative

_div_id and _gradient_id are mentioned more than once in proximal_operators.py so I did not look at them yet

Reply:
actually now I have renamed them too